python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/graphql/getChartDetails.graphql

```diff
@@ -0,0 +1,41 @@
+query getChartDetails($urn: String!) {
+  chart(urn: $urn) {
+    urn
+    type
+    platform {
+      name
+    }
+    ownership {
```

Comment:
I think we will need this to populate the side panel with contact details but it's missing from ChartMetadata at the moment. I think it's ok to leave this for now as it's not quite right for the other details pages either. I think we can fix this once we've got roles and ownership types properly defined?